### PR TITLE
Add integration codeowners to sidebar

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

To celebrate our codeowners and to put them more in the spot (they deserve to be for their awesome work). I've added them to the sidebar:

With codeowners:

image

No codeowners:

image

Core maintained integration:

image

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by c0ffeeca7 — DISMISSED** [review]
Nice! Thank you, @frenck :+1:

File: source/_includes/asides/component_navigation.html [hunk 1]
```diff
@@ -73,5 +73,36 @@
...
+        {% for codeowner in page.ha_codeowners %}
+            {% if codeowner contains "@home-assistant/" %}
+                {% assign ha_project = true %}
+            {% endif %}
+        {% endfor %}
+        
+        {% if ha_project %}
+            This integration is being maintained by the Home Assistant project.
+        {% else %}
+          We are incredibly grateful to the following contributors who currently maintain this integration:<br />
```
Comment (ludeeus) [inline source/_includes/asides/component_navigation.html]:
```suggestion
          We are incredibly grateful to the following contributors who currently maintain this integration:
```
Should do a single here to match the section above it.

Comment (frenck) [reply source/_includes/asides/component_navigation.html]: It looks really weird if we do that... As in, the list is almost embedded in the above text at that point.

**Review by c0ffeeca7 — DISMISSED** [review]

File: source/_includes/asides/component_navigation.html (hunk 1 above)
Comment (ludeeus) [reply source/_includes/asides/component_navigation.html]: Meh... yeah that did not look any better.
For it to be natural with the other sections, it need more defined sections and not a section within another section, like this:
```diff
--- a/source/_includes/asides/component_navigation.html
+++ b/source/_includes/asides/component_navigation.html
@@ -77,7 +77,6 @@
 </section>
 
 <section class="aside-module grid__item one-whole lap-one-half">
-  <div class='section'>
     <h1 class="title delta">Codeowners</h1>
     {% if page.ha_codeowners %}
         {% assign ha_project = false %}
@@ -88,10 +87,13 @@
         {% endfor %}
         
         {% if ha_project %}
+          <div class='section'>
             This integration is being maintained by the Home Assistant project.
+          </div>
         {% else %}
-          We are incredibly grateful to the following contributors who currently maintain this integration:<br />
-          <br />
+        <div class='section'>
+          We are incredibly grateful to the following contributors who currently maintain this integration:
+          </div>
           <div class="section">
             {%- for codeowner in page.ha_codeowners -%}
             {%- assign clean_codeowner = codeowner | replace: "@", "" -%}
@@ -100,9 +102,10 @@
           </div>
         {% endif %}
     {% else %}
-        This integration is community maintained.<br />
-        If you are a developer and would like to help, feel free to contribute!
+        <div class='section'>
+          This integration is community maintained.<br />
+          If you are a developer and would like to help, feel free to contribute!
+        </div>
     {% endif %}
-  </div>
 </section>
 ```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_includes/asides/component_navigation.html (hunk 1 above)
Comment (frenck) [reply source/_includes/asides/component_navigation.html]: Thanks! much better 👍

**Review by ludeeus — APPROVED** [review]
Awesome! 🎉

**Pull request merged**